docs/src/main/sphinx/release/release-360.md

```diff
@@ -0,0 +1,72 @@
+# Release 360 (XXX Jul 2021)
+
+## General
+
+* Improve support for correlated subqueries with GROUP BY or LIMIT and 
+  complex correlated filter conditions. ({issue}`8554`)
+* Disallow `DROP SCHEMA` if the schema is not empty. ({issue}`8660`)
+* Fix query failure when there is a constant predicate on some
+  column `col` (e.g `col=1`), followed by  `ORDER BY col` and `LIMIT`. ({issue}`8535`)
+* Fix `SHOW CREATE SCHEMA` failure. ({issue}`8598`)
+* Fix query failure when running `SHOW CREATE SCHEMA information_schema`. ({issue}`8600`)
+* Improve performance of `WHERE` clause evaluation. ({issue}`8624`)
+* Reduce coordinator network load. ({issue}`8460`)
+* Improve query performance by sending collected dynamic filters from coordinator to workers. ({issue}`5183`)
+* Improve performance of inequality joins where join condition sides have different type. ({issue}`8500`)
+* Improve performance of `IN (<subquery>)` expressions. ({issue}`8639`)
+
+## Security
+
+* Add support for automatic configuration of TLS for {doc}`/security/internal-communication`. This removes 
+  the need to provision per-worker TLS certificates. ({issue}`7954`)
+  
+## CLI
+
+* Fix auto completion when pressing the tab button. ({issue}`8529`)
+
+## Hive connector
+                                
+* Fix data corruption when performing `UPDATE` or `INSERT` on ORC ACID transactional table. ({issue}`8268`, {issue}`8452`) 
+  
+## Iceberg connector
+
+* Add support for Trino views. ({issue}`8540`)
+* Prevent incorrect query results by failing a query when Iceberg table has row-level deletes. ({issue}`8450`)
+* Fix query failure when joining with a bucketed Iceberg table. ({issue}`7502`)
+* Fix query failure when showing stats for a bucketed Iceberg table. ({issue}`8616`)
+* Fix query failure when joining with a partitioned table that has structural columns (`array`, `map` or `row`). ({issue}`8647`)
+* Fix failures for queries that write tables in Parquet format. ({issue}`5201`)
+* Improve query planning time by reducing calls to the metastore. ({issue}`8676`, {issue}`8689`)
+
+## MongoDB connector
+
+* Add support for `json` type. ({issue}`8352`)
+* Support reading MongoDB `DBRef` type. ({issue}`3134`)
+
+## Phoenix connector
+
+* Improve performance of `ORDER BY ... LIMIT` queries on sorted data for Phoenix 5. ({issue}`8171`)
+
+## Other connectors
+                                           
+This change applies to the ClickHouse, Druid, MemSQL, MySQL, Oracle, Phoenix, PostgreSQL, Redshift, and SQLServer connectors:
```

Comment:
@martint You can remove Druid from this list. It doesn't support writes yet so the config is meaningless for it.

Reply:
Once this is done you can merge the `insert.non-transactional-insert.enabled` note under the same heading since the list of connectors is same.

Reply:
Yup. I'm going to unroll it into each connector anyway, so not a big deal.